jk-barcode/projects/jk-barcode/ng-package.json

```diff
@@ -3,5 +3,6 @@
   "dest": "../../dist/jk-barcode",
   "lib": {
     "entryFile": "src/public-api.ts"
-  }
-}
\ No newline at end of file
+  },
+  "allowedNonPeerDependencies": ["bwip-js"]
```

Comment:
This is not recommended. But than your demo app will definetly need to install bwip-js, even when installed from npm. 

If thats fine for u, drop this and move bwip-js from dependencies to peerDependencies in the package.json that gets shipped